react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt

```diff
@@ -0,0 +1,47 @@
+package com.revenuecat.purchases.react.ui
+
+import android.annotation.SuppressLint
+import androidx.core.view.children
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.revenuecat.purchases.ui.revenuecatui.ExperimentalPreviewRevenueCatUIPurchasesAPI
+import com.revenuecat.purchases.ui.revenuecatui.views.PaywallFooterView
+
+@OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
+internal class PaywallFooterViewManager : SimpleViewManager<PaywallFooterView>() {
+    override fun getName(): String {
+        return "RCPaywallFooterView"
+    }
+
+    @SuppressLint("UnsafeOptInUsageError")
+    override fun createViewInstance(themedReactContext: ThemedReactContext): PaywallFooterView {
+        val paywallFooterView: PaywallFooterView = object : PaywallFooterView(themedReactContext) {
+            public override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
```

Comment:
This will crash until https://github.com/RevenueCat/purchases-android/pull/1541 is released